src/components/CardsContainer.tsx

```diff
@@ -1,28 +1,83 @@
 'use client'
 
 import UserCard from './UserCard'
+import { useState, useEffect } from 'react'
+
 import { namesArray } from '@/utils/sampleData'
 import { sealifeImages } from '@/utils/imageLists'
+// import ChatQuestionOutline from './icons/JellyFishOutlineIcon'
+import { useAllUsersPointsListener } from '@/services/socket'
+import { useSocketListener } from '@/services/socket'
+import type { ListenerRes } from '@/services/socket'
 
 export default function CardsContainer() {
-	const sampleNames = namesArray.slice(10, 30)
+	const [usersPointsData, setUsersPointsData] = useState<ListenerRes[]>([])
+	const [showStoryPoints, setShowStoryPoints] = useState(false)
+	// const sampleNames = namesArray.slice(0, 9)
+	const sampleNames: string[] = []
 	const storyPointsArray = [0, 1, 2, 3, 5, 8, 13]
 
+	const allUsersPoints = useAllUsersPointsListener('all-users-story-points')
+
+	// the line below creates an infinite loop
+	// allUsersPoints && setUsersPointsData(allUsersPoints?.message)
+
+	const showStoryPointsListener = useSocketListener(
+		'show-disable-reset-points',
+	)
+
+	useEffect(() => {
+		if (!allUsersPoints) return
+		const listenerMessages = allUsersPoints.message
+		console.log(
+			'%c>>> listenerMessages:',
+			'color: yellow',
+			listenerMessages,
+		)
+		setUsersPointsData(listenerMessages)
+	}, [allUsersPoints])
+
+	useEffect(() => {
```

Comment:
todo: This 2 `useEffect` can be replaced by the socket callback pattern we spoke in our call :)

Reply:
DONE